modules/storage-persistent-storage-overview.adoc

```diff
@@ -8,38 +8,19 @@
 [id=persistent-storage-overview_{context}]
 = Persistent storage overview
 
-Managing storage is a distinct problem from managing compute resources.
-{product-title} uses the Kubernetes persistent volume (PV) framework to
-allow cluster administrators to provision persistent storage for a cluster.
-Developers can use persistent volume claims (PVCs) to request PV resources
+Managing storage is a distinct problem from managing compute resources. {product-title} uses the Kubernetes persistent volume (PV) framework to allow cluster administrators to provision persistent storage for a cluster. Developers can use persistent volume claims (PVCs) to request PV resources
 without having specific knowledge of the underlying storage infrastructure.
 
-PVCs are specific to a project, and are created and used by developers as
-a means to use a PV. PV resources on their own are not scoped to any
-single project; they can be shared across the entire {product-title}
-cluster and claimed from any project. After a PV is bound to a PVC,
-that PV can not then be bound to additional PVCs. This has the effect of
-scoping a bound PV to a single namespace, that of the binding project.
+PVCs are specific to a project, and are created and used by developers as a means to use a PV. PV resources on their own are not scoped to any single project; they can be shared across the entire {product-title} cluster and claimed from any project. After a PV is bound to a PVC, that PV can not then be bound to additional PVCs. This has the effect of scoping a bound PV to a single namespace, that of the binding project.
```

Comment:
```suggestion
PVCs are specific to a namespace, and are created and used by developers as a means to use a PV. PV resources on their own are not scoped to any single namespace; they can be shared across the entire {product-title} cluster and claimed from any namespace. After a PV is bound to a PVC, that PV can not then be bound to additional PVCs. This has the effect of scoping a bound PV to a single namespace.
```

MicroShift does not support the project api. Using it interchangeably with namespace might cause some confusion for readers.